app/src/main/java/com/android/sample/ui/listActivities/ViewListActivities.kt

```diff
@@ -190,7 +223,8 @@ fun ActivityCard(
     navigationActions: NavigationActions,
     listActivitiesViewModel: ListActivitiesViewModel,
     profileViewModel: ProfileViewModel,
-    profile: User?
+    profile: User?,
+    distance: Float? = null
```

Comment:
Great you gave it a default value ! It helped you not change code in every place ActivityCard is used (such as the tests).